### DISCUSSION TITLE
reply to comment

### DISCUSSION THREAD

**Discussion by minyoung90** [dolanmiu/docx#2051] [Q&A] [answered]
already asked before https://github.com/dolanmiu/docx/discussions/1868 https://github.com/dolanmiu/docx/discussions/1950
but, there is no answer. so I open again.

Making one single comment is very easy, but no idea like picture below 
![image](https://user-images.githubusercontent.com/17244668/228539785-d8032a0c-70c4-4161-ac6e-d72f22cedebd.png)

"docx" can make like this?

**Comment by dolanmiu** [comment, marked as answer]
Not possible to reply just yet I think, I need to investigate

But adding multiple paragraphs is supported, and I suspect it would be something like this, bear with me

**Reply by dolanmiu** [reply]
Replying is not possible.

I did some research, and it is a Microsoft Word only feature. This will not work in other Word Processors
